pyo3-build-config/src/impl_.rs

```diff
@@ -669,6 +691,7 @@ impl Display for PythonImplementation {
         match self {
             PythonImplementation::CPython => write!(f, "CPython"),
             PythonImplementation::PyPy => write!(f, "PyPy"),
+            PythonImplementation::GraalPy => write!(f, "GraalVM"),
```

Comment:
Is the use of "GraalVM" instead of "GraalPy" here and below in `FromStr` deliberate?

Reply:
Yes, GraalPy report's that string for `platform.python_implementation()`